doc/setup.md

```diff
@@ -79,15 +79,32 @@ sudo service postgresql restart
 - Extract the binary to somewhere on your path
 - Ensure it runs with `geckodriver --version`
 
+4. make the dependencies using
```

Comment:
This is done automatically upon first launch of skyportal.

Reply:
Ok I will remove it.